src/renderer/deviceMessages.ts

```diff
@@ -33,4 +33,14 @@ Full Changelog: https://github.com/deltachat/deltachat-desktop/blob/master/CHANG
 
 Full Changelog: https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md#1_38_0`
   )
+
+  BackendRemote.rpc.addDeviceMessage(
+    accountId,
+    'changelog-version-1.39.0-version0',
+    `What's new in 1.39.0?
+
+✨ We fixed many small bugs and improved stability
+
+Full Changelog: https://github.com/deltachat/deltachat-desktop/blob/master/CHANGELOG.md#1_39_0`
+  )
```

Comment:
I personally don't like this message. We could at least tell them what fixes we did. Additionally, shouldn't this be multilingual and from transifex?

Reply:
there is nothing very big in this release, so better no message